### PR TITLE
Re-org Simulation section and remove duplications

### PR DESCRIPTION
**Pull request by Hwurzburg**
had to run SITL after not running for several months and was very hard for me to find the information I needed (some was missing, I will fix that next)....lots of duplicate TOC entries...
so this is to simplify the drill down to what someone is looking for rather than a long list of topics...

no substance changes, just re-organization...makes its quick to find what you have forgotten or easy(er) for a first time user....

next , I am going to add stuff to a couple of pages that I had to get Tridges help to run QuadPlane yesterday while its fresh in memory

### CODE REVIEW

File: dev/source/docs/simulation-2.rst [hunk 1]
```diff
@@ -16,12 +16,14 @@ The most commonly used simulators are:
 
 -  :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` is the simulator most commonly used by developers. It is a simple simulator that is built within all SITL builds of ArduPilot. It is used by the :ref:`autotester <the-ardupilot-autotest-framework>` and other simulators below are actually built on top of SITL
 -  :ref:`Gazebo <using-gazebo-simulator-with-sitl>` is the official DARPA virtual robotics simulator
--  :ref:`XPlane-10 <sitl-with-xplane>` a commercial flight simulator with a rich 3D interface
+-  :ref:`XPlane-10 <sitl-with-xplane>` a commercial flight simulator with a rich 3D interface   
+-  :ref:`X-Plane-10 Soaring<soaring-sitl-with-xplane>` soaring on XPlane-10
```
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst]: Minor inconsistency issue with XPlane vs X-Plane

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst [hunk 2]
```diff
@@ -47,8 +47,8 @@ Running SITL
 ============
 
 The ArduPilot SITL environment has been developed to run natively on both
-Linux and Windows. For instructions see :ref:`Setting up SITL on Linux <setting-up-sitl-on-linux>` and :ref:`Setting up SITL on Windows <sitl-native-on-windows>`
-for more information.
+Linux and Windows. For setup instructions see :ref:`Setting Up SITL <SITL-setup-landingpage>`
+for more information. For starting SITL for a particular vehicle see :ref:`Evoking SITL <using-sitl>`, while environment setup and how to use SITL is explained in :ref:`Using SITL <using-sitl-for-ardupilot-testing>`
```
Comment (peterbarker) [inline dev/source/docs/sitl-simulator-software-in-the-loop.rst]:
```suggestion
for more information. For starting SITL for a particular vehicle see :ref:`Invoking SITL <using-sitl>`, while environment setup and how to use SITL is explained in :ref:`Using SITL <using-sitl-for-ardupilot-testing>`
```

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst [hunk 3]
```diff
@@ -64,18 +64,7 @@ port numbers depending on your environment.
...
-    Balance Bot SITL/MAVProxy Tutorial <balance_bot-sitl-mavproxy-tutorial>
-    SITL Advanced Testing <using-sitl-for-ardupilot-testing>
-    Using Gazebo Simulator with SITL <using-gazebo-simulator-with-sitl>
-    Using Last_letter Simulator with SITL <using-last_letter-as-an-external-sitl-simulator>
-    Using the CRRCSim simulator <simulation-2sitl-simulator-software-in-the-loopusing-using-the-crrcsim-simulator>
-    Using X-Plane 10 with SITL <sitl-with-xplane>
-    Soaring SITL with X-Plane <soaring-sitl-with-xplane>
-    Soaring SITL with Silent Wings <soaring-sitl-with-silentwings>
+    Setting Up SITL <SITL-setup-landingpage>
+    Evoking SITL <using-sitl>
```
Comment (peterbarker) [inline dev/source/docs/sitl-simulator-software-in-the-loop.rst]:
```suggestion
    Invoking SITL <using-sitl>
```

File: dev/source/docs/simulation-2.rst (hunk 1 above)
Comment (peterbarker) [inline dev/source/docs/simulation-2.rst]:
```suggestion
-  :ref:`XPlane-10 Soaring<soaring-sitl-with-xplane>` soaring on XPlane-10
```

File: dev/source/docs/using-sitl-for-ardupilot-testing.rst [hunk 4]
```diff
@@ -1,8 +1,8 @@
 .. _using-sitl-for-ardupilot-testing:
 
-=====================
-SITL Advanced Testing
-=====================
+==========
+Using SITL
```
Comment (peterbarker) [inline dev/source/docs/using-sitl-for-ardupilot-testing.rst]: Don't we now have two pages with the same title?

Is one of these pages supposed to essentially be a "quick-start" where the other is a "how to use it to properly test ArduPilot"?

File: dev/source/docs/simulation-2.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst]: mea copa

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst (hunk 2 above)
Comment (Hwurzburg) [reply dev/source/docs/sitl-simulator-software-in-the-loop.rst]: yep will change

File: dev/source/docs/sitl-simulator-software-in-the-loop.rst (hunk 3 above)
Comment (Hwurzburg) [reply dev/source/docs/sitl-simulator-software-in-the-loop.rst]: yep

File: dev/source/docs/simulation-2.rst (hunk 1 above)
Comment (Hwurzburg) [reply dev/source/docs/simulation-2.rst]: mea copa

File: dev/source/docs/using-sitl-for-ardupilot-testing.rst (hunk 4 above)
Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: yeah as I was changing names I missed this....should be setting up, starting, using....will change...and Invoke is too expensive a word..

**Pull request merged**